Doc/glossary.rst

```diff
@@ -320,6 +320,21 @@ Glossary
       A module written in C or C++, using Python's C API to interact with the
       core and with user code.
 
+   f-string
+      Short for :dfn:`formatted string literal`, which is string literal
+      that is prefixed with ``'f'`` or ``'F'``.  f-strings contain
```

Comment:
Hmm.. I followed PEP 498 where it is not capitalized. https://github.com/python/peps/blob/master/pep-0498.txt#L29

Should the PEP be updated?